packages/react-magma-dom/src/components/Table/TablePagination.tsx

```diff
@@ -135,10 +130,10 @@ export const TablePagination = React.forwardRef<
   TablePaginationProps
 >((props, ref) => {
   const {
+    dropdownDropDirection,
     testId,
     defaultPage,
     defaultRowsPerPage = 10,
-    dropdownDropDirection = DropdownDropDirection.up,
```

Comment:
Should we keep this default in the meantime?

Reply:
I think you would want the Native Select to be the default option and if we provide a default value to dropdownDropDirection then the Dropdown Component would be used.